core/domain/skill_domain.py

```diff
@@ -642,6 +642,15 @@ def add_misconception(self, misconception_dict):
             misconception_dict['id'])
 
     def get_incremented_misconception_id(self, misconception_id):
+        """Returns the incremented misconception id.
+
+        Args:
+            misconception_id: str. The id of the misconception to be
+                incremented.
+
+        Returns:
+            str. The incremented misconception id.
+        """
         return misconception_id + 1
```

Comment:
Hi @seanlip, I found a catch here.  `misconception_id` is of the type `str` then, how can we add integer value to the string? We need explicit type casting here. Should I change it to- `str( int( misconception_id) + 1)`?

/cc @aks681

Reply:
Actually, misconception_id should be int. Yeah, many docstrings mention it as str, but the main docstring at class Misconception mentions it to be int. 
This is because these would be given the value present in next_misconception_id in class Skill, which is int.

Reply:
Thanks @aks681! @vibhor98 can we fix these docstrings too please?

Reply:
Ok, I'll change the docstrings everywhere else also.

Reply:
Done.